kythe/cxx/indexer/cxx/testdata/objc/tvar_generics/generic_bound.m

```diff
@@ -19,10 +19,10 @@ @implementation O2
 // No source range defines BoxDecl since this is a generic type.
 //- @Type defines/binding TypeVar
 //- @Box defines/binding BoxAbs
-//- TypeVar.node/kind absvar
+//- TypeVar.node/kind tvar
 //- BoxDecl childof BoxAbs
 //- BoxAbs.node/kind abs
```

Comment:
Shouldn't we no longer emit the `abs` nodes in this case?

Reply:
In the fullness of time, yes. As work proceeds on implementing the new model I'll continue to update the forked tests. Right now only tvar/tparam are implemented.